kapeta.yml

```diff
@@ -0,0 +1,10 @@
+kind: core/language-target
+metadata:
+  name: kapeta/language-target-docker
+  title: Docker
+  description: Use your existing code that is packaged as a dockerfile
```

Comment:
maybe this somehow should indicate that it's for non kapeta projects... so people don't think they should use this if they want a project with a docker file